review/src/NoDuplicatePorts.elm

```diff
@@ -0,0 +1,113 @@
+module NoDuplicatePorts exposing (rule)
+
+{-| Highlights duplicate port names, which will break the Elm compiler
+-}
+
+import Dict exposing (Dict)
+import Elm.Syntax.Declaration as Declaration exposing (Declaration)
+import Elm.Syntax.ModuleName exposing (ModuleName)
+import Elm.Syntax.Node as Node exposing (Node)
+import Elm.Syntax.Range exposing (Range)
+import Review.Rule as Rule exposing (Direction, Error, Rule)
+
+
+error : { message : String, details : List String }
+error =
+    { message = "Ensure that port names are unique across your project."
+    , details = [ "This port has been defined elsewhere." ]
```

Comment:
Here, I'd re-explain what the problem is like in the `rule`'s documentation, but with less depth.

Something like this could be sufficient: "Having ports with the same name leads to runtime errors when they get accessed from JavaScript" (I don't remember exactly what the problem is)